lua/femaco/edit.lua

```diff
@@ -44,21 +44,28 @@ local get_match_text = function(match, bufnr)
 end
 
 local parse_match = function(match)
-    local language = match.language or match._lang
-    if language == nil then
-      for lang, val in pairs(match) do
-        return {
-          lang = lang,
-          content_match = val,
-        }
-      end
-    else
+  local language = match.language or match._lang
+  if language == nil then
```

Comment:
[nit] maybe just for cleanness this could be written:
```lua
if language then
  return ...
end
if match.injection then
  return ...
end
for lang, val, .. do
end
```
What do you think?

Reply:
I think it is an good idea, and I have fixed it.